packages/pageheader/README.md

```diff
@@ -1,36 +1,24 @@
 # PageHeader
 
-The PageHeader component provides information and actions for the current screen. Designers and developers using a PageHeader can use their judgement to decide whether the PageHeader scrolls with the page content, or is stuck at the top.
+The PageHeader component is used to provide information and actions for the current screen. It can be customized to either scroll with the page content or remain fixed at the top.
```

Comment:
I would keep this first sentence in active voice like it was. The second sentence maybe "Customize it to either scroll..."